src/thermo/PengRobinson.cpp

```diff
@@ -780,4 +780,18 @@ int PengRobinson::solveCubic(double T, double pres, double a, double b, double a
                                          an, bn, cn, dn, tc, vc);
 }
 
+AnyValue PengRobinson::compute_extra_method(const std::string& name)
+{
+    if(name == "aAlpha_mix") {
```

Comment:
```suggestion
    if (name == "aAlpha_mix") {
```
Missing whitespace after `if`